packages/manager/modules/pci/src/projects/project/storages/databases/database/topics/topics.html

```diff
@@ -0,0 +1,100 @@
+<div class="container-fluid px-0 mt-3">
+    <!-- error messages -->
+    <cui-message-container
+        data-messages="$ctrl.messages"
+    ></cui-message-container>
+
+    <h2 data-translate="pci_databases_topics_tab_title"></h2>
+
+    <oui-datagrid class="mt-5" data-page-size="10" data-rows="$ctrl.topicsList">
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_name' | translate"
+            data-property="name"
+            data-sortable="asc"
+            data-type="string"
+            data-searchable
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_partitions' | translate"
+            data-property="partitions"
+            data-sortable="asc"
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_replication' | translate"
+            data-property="replication"
+            data-sortable="asc"
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_minInsyncReplicas' | translate"
+            data-property="minInsyncReplicas"
+            data-sortable="asc"
+            data-filterable
+        ></oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_retention_hours' | translate"
+            data-property="retentionHours"
+            data-sortable="asc"
+            data-filterable
+        >
+            <span
+                ng-if="$row.retentionHours >= 0"
+                data-ng-bind="$row.retentionHours"
+            ></span>
+            <span
+                ng-if="$row.retentionHours < 0"
+                data-ng-bind=":: 'pci_databases_topics_tab_unlimited' | translate"
+            ></span>
+        </oui-datagrid-column>
+        <oui-datagrid-column
+            data-title=":: 'pci_databases_topics_tab_retention_bytes' | translate"
+            data-property="retentionBytes"
+            data-sortable="asc"
+            data-filterable
+        >
+            <span
+                ng-if="$row.retentionBytes >= 0"
+                data-ng-bind="$row.retentionBytes"
```

Comment:
If it is not going to change dymanically

```suggestion
                data-ng-bind="::$row.retentionBytes"
```

Reply:
Done here: https://github.com/ovh/manager/pull/5488/commits/45d16da89ff2c70503a9a111c3fd5996a5eecbe1